src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/DisablePrivateReflectionAttribute.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace System.Runtime.CompilerServices
 {
+    [Obsolete(Obsoletions.EscapeUriStringMessage, DiagnosticId = Obsoletions.EscapeUriStringDiagId, UrlFormat = Obsoletions.SharedUrlFormat)]
```

Comment:
This needs a new message and new diagnostic ID for this particular obsoletion added to that file, and then those new values used here.  It shouldn't use the existing ones, which aren't relevant to this API.